modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/source/ExtensionMessageSource.java

```diff
@@ -149,6 +148,7 @@
   private SourceAdapter sourceAdapter;
   private RetryPolicyTemplate retryPolicyTemplate;
   private Scheduler retryScheduler;
+  private Scheduler reconnectScheduler;
```

Comment:
you already have a retry scheduler. Why not reuse that one?

Reply:
Done